lsp/src/import.ml

```diff
@@ -18,6 +18,8 @@ module String = struct
 
   let index = index_opt
 
+  let is_empty s = length s = 0
```

Comment:
Btw, this function can also be written as `= ""` with equivalent performance. Additionally `not (is_empty s)` is just `<> ""`.

Reply:
Interesting, that's how I initially wrote it and then I saw the `stdune` implementation: https://github.com/ocaml/dune/blob/main/otherlibs/stdune/string.ml#L43

Good to know that it doesn't make any difference! :)